java/main/src/main/java/com/saucelabs/saucebindings/options/SauceOptions.java

```diff
@@ -206,6 +212,23 @@ public Map<Timeouts, Integer> getTimeouts() {
     }
   }
 
+  public SauceOptions copy() {
```

Comment:
I still miss why this is needed. Are we doing the same thing in Selenium?

Reply:
Selenium capabilities classes implement Serializable which means you can implement a copy feature easily if you want (which I didn't know before this week)

I tried a bunch of other things first and ran into one issue after another; I settled on this because it guarantees its not a shallow copy without my having to write dozens of tests, and it is overall a very simple implementation